setup.py

```diff
@@ -82,7 +43,6 @@ def get_setup_version(reponame):
     url='http://ioam.github.com/param/',
     packages=["param","numbergen"],
     provides=["param","numbergen"],
-    package_data = {'pkg_bundle': ['.version']},
```

Comment:
It's already in MANIFEST.in; combined with `include_package_data=True`, this is enough to ensure .version ends up in packages, and that .version ends up in the package when installed.

(Note that things have been working even though it says `pkg_bundle` above rather than `param`...)

Reply:
Ok. I've never been clear on this so I tend to  try everything to make sure!